connectivity/ft_connectivity_wpli.m

```diff
@@ -66,7 +66,11 @@
   else
     wpli     = outsum./outsumW; % estimator of E(Im(X))/E(|Im(X)|)
   end
-  wpli = reshape(wpli,siz(2:end)); % remove the first singular dimension
+  if (length(siz) == 2)
+    wpli = reshape(wpli,[ siz(2:end) 1 ] );
+  else
```

Comment:
Rather than using the if-else statement, I'd recommend to change line 56 into siz = [size(input) 1]; 
This will always work.